src/main/resources/templates/fragments/header.html

```diff
@@ -51,16 +51,20 @@
                             <a class="dropdown-item" href="/about">About</a>
                         </ul>
                     </li>
-                    <div class="theme-toggle navbar-text">
-                        <label>
-                            <span class="visually-hidden">Light</span>
-                            <input type="radio" value="light" name="theme">
-                            <span class="checkmark" th:attr="data-cy='light-mode-radio'">☀️</span>
-                        </label>
-                        <label>
-                            <span class="visually-hidden">Dark</span>
-                            <input type="radio" value="dark" name="theme">
-                            <span class="checkmark" th:attr="data-cy='dark-mode-radio'">🌜</span>
```

Comment:
the attributes are required for people who have trouble seeing and are required for the cypress tests to run. Can you add the data-cy attrs to your new implementation please?
And the cypress tests need updating to use the new labels to be able to switch themes